src/app/submission/pages/submission.component.ts

```diff
@@ -446,5 +445,7 @@ export class SubmissionComponent implements OnInit, OnDestroy {
     )
   }
 
-
+  addToManifest(key: string) {
```

Comment:
It might be better to patch the manifest. The count will be off again if you refresh the page, isn't it?

Reply:
Indeed this is only correcting the count until the page is refreshed, by then the summary will have been patched by the staging manager.

Reply:
I don't think staging manager is patching the summary 🤔

Reply:
@MightyAx not sure if you've seen my reply ^

Reply:
Need a new ticket for this as ingest-core work

Reply:
New ticket https://github.com/ebi-ait/dcp-ingest-central/issues/738